src/browser/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+#  Copyright (C) 2017 Sami Vänttinen <sami.vanttinen@protonmail.com>
+#  Copyright (C) 2017 KeePassXC Team <team@keepassxc.org>
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+if(WITH_XC_BROWSER)
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+
+    find_package(sodium 1.0.12 REQUIRED)
+    if(SODIUM_FOUND)
+        include_directories(${sodium_INCLUDE_DIR})
+        set(sodium_INCLUDE_EXTRAS "${sodium_INCLUDE_DIR}/sodium")
+        include_directories(${sodium_INCLUDE_EXTRAS})
+        set(pkg_config_libs_private "${pkg_config_libs_private} -lsodium")
+    else()
+        message(FATAL_ERROR "libsodium is not installed. Install it, then run CMake again")
+    endif()
```

Comment:
Remove everything from if(SODIUM_FOUND) to endif(), it is simply not needed at all. The find_package method handles all of this for you.

Reply:
Done.